react-challenge.html

```diff
@@ -48,10 +49,14 @@ <h3>Mood Options:</h3>
                 { this.state.currentMood }
               </div>
 
-              <button onClick={ handleMoodOptions }>
+              <button onClick={ this.handleMoodOptions }>
```

Comment:
So you managed to get most of the problems! But i'm wondering, when you previewed this file and clicked this button after all of your changes ,did it work?